scripts/src/main/resources/scripts/command/jmc

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup version help"
+  fi
+  exit
+fi
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+JMC_HOME="${DEVON_IDE_HOME}/software/jmc"
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
+
+# Call jmc with specified arguments.
+function doRun() {
+  if [ -z "$(ls -A "${JMC_HOME}")" ]
+  then
+    doSetup silent
+  fi
+  doRunCommand "${JMC_HOME}/jmc ${*} &"
+}
+
+# JDK Mission Control installation
+function doSetup() {
+  local install_dir="${DEVON_IDE_HOME}/updates/install/jmc"
+  doInstall "jmc" "${JMC_VERSION}" "${1}" "" "${install_dir}"
```

Comment:
why dont you use `${JMC_HOME}` instead of `${install_dir}`?
You are installing to a "temporary" directory and after that you are moving individual files from there to the final directory. The inner structure of the JMC download archive may change in the future what will then break this code.
If we want to have the `JDK Mission Control` content on the toplevel directory so the PATH gets computed correctly, you could afterwards move all the contents of the `JDK Mission Control` folder (if it exists) to its parent (`${JMC_HOME}`).
So if one day they change their download archive to a more common structure (e.g. with toplevel `bin` folder), our code will still work. Also you do not have to move `legal` and `.devon.software.version` then.

Reply:
Thank you for the suggestion, I initially build it that way, as it was done so in other scripts, I changed it accordingly now and it works the same way